tools/shared/src/wasmatic.rs

```diff
@@ -58,6 +64,7 @@ pub async fn deploy(
         "envs": envs,
         "testable": testable,
     });
+    println!("body {:?}", body);
```

Comment:
is this `println` meant to be left in?

Reply:
It was for debugging, can be removed.
I will ditch this branch and fix it wasmatic side